sw/device/tests/meson.build

```diff
@@ -835,8 +835,7 @@ foreach sw_test_name, sw_test_info : sw_rom_ext_tests
         rom_ext_slot_libs['rom_ext_slot_a'],
         device_lib,
         sw_test_info['library'],
-        sw_lib_irq_handlers,
-        sw_lib_testing_test_main,
+        sw_lib_testing_ottf,
```

Comment:
I don't think we have a CI test that uses images created in this loop but we are planning to add one as soon as fpga bitstream splicing issue for the mask rom is resolved. @moidx

Reply:
I think @moidx is working on this (#9450) so hopefully should be able to test this soon too.